wasmtestreport.py

```diff
@@ -632,13 +633,23 @@ def main():
     should_generate_html = True
     pre_test_only = args.pre_test_only
     clean_testfiles = args.clean_testfiles
+    clean_results = args.clean_results
+
+    if clean_results:
+        if os.path.isfile(output_file):
+            os.remove(output_file)
+        if os.path.isfile(output_html_file):
+            os.remove(output_html_file)
+        print(Path(LIND_FS_ROOT))
+        for file in Path(LIND_FS_ROOT).iterdir():
+            file.unlink()
```

Comment:
If, in the future, this changes from just being files in this location to folders as well, will this work? perhaps each item could be checked to see if it is a file or a folder and removed appropriately depending on what it is?